ygot/struct_validation_map.go

```diff
@@ -527,8 +545,22 @@ func DeepCopy(s GoStruct) (GoStruct, error) {
 	return n.Interface().(GoStruct), nil
 }
 
+// fieldOverwriteEnabled looks for the presence of MergeOverwriteExistingFields
+// from the MergeOpt slice provided.
+// In the case that there are multiple MergeOpt structs within the slice,
```

Comment:
(nit) This sentence seems to be irrelevant now (since we just return a `bool`) can you remove it?